Dashboard default credentials substitution in kong.yml (#16679)
supabase/supabase: apps/docs/pages/guides/self-hosting/docker.mdx

### Dashboard Authentication

The dashboard is protected with Basic Authentication. The default user and password MUST be updated before using Supabase in production.
Update the following values in the `.env` file:
 - `DASHBOARD_USERNAME`: The default username for the Dashboard 
 - `DASHBOARD_PASSWORD`: The default password for the Dashboard 

> You can Also update the `./docker/volumes/api/kong.yml` file with your own credentials. For example:

Reviewer: I think we don't need this part

> You can Also update the `./docker/volumes/api/kong.yml` file with your own credentials. For example:

Reviewer: ```suggestion
You can also add more credentials in `./docker/volumes/api/kong.yml`. For example:
```
